persistence-modules/spring-boot-persistence-3/src/main/java/com/baeldung/countrows/entity/Permission.java

```diff
@@ -0,0 +1,39 @@
+package com.baeldung.countrows.entity;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name="PERMISSIONS")
+public class Permission {
+
+    @Id
+    @GeneratedValue(strategy= GenerationType.SEQUENCE, generator = "permissions_id_sq")
+    @SequenceGenerator(name = "permissions_id_sq", sequenceName = "permissions_id_sq", allocationSize = 1)
+    private int id;
```

Comment:
Not used anywhere, we can remove it.

Reply:
this is primary_key and my understanding is that we cannot remove it